modules/boot/src/save_specials.cpp

```diff
@@ -79,6 +79,10 @@ KEEP_FUNC void SaveMngSpecial_SetLayer3() {
     g_dComIfG_gameInfo.play.mNextStage.setLayer(3);
 }
 
+KEEP_FUNC void SaveMngSpecial_SetLayer8() {
+    g_dComIfG_gameInfo.play.mNextStage.setLayer(8);
+}
+
```

Comment:
this was my mistake previously-- can you change all of these "SetLayerX" function from `KEEP_FUNC` to `inline` instead?

Reply:
nevermind actually this is right, ignore. 
I forgot these get called by the saves themselves externally